frontend/src/views/GShootList.vue

```diff
@@ -281,20 +282,21 @@ export default {
     })
   },
   beforeRouteUpdate (to, from, next) {
-    this.shootSearch = null
+    this.resetShootSearch()
     this.updateTableSettings()
     this.focusModeInternal = false
     next()
   },
   beforeRouteLeave (to, from, next) {
     this.cachedItems = this.shootList.slice(0)
-    this.shootSearch = null
+    this.resetShootSearch()
     this.focusModeInternal = false
     next()
   },
   data () {
     return {
       shootSearch: '',
+      delayedShootSearch: '',
```

Comment:
Why do we need two properties? If we need it the name should be `debouncedShootSearch` to make it more clear what is is.

Reply:
✅  as discussed